File: templates/Uwp/Pages/ListDetail.Prism/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the method `GetListDetailDataAsync()` to the SampleDataService of your project.
//**}
namespace Param_RootNamespace.Core.Services
{
    public class SampleDataService : ISampleDataService
    {
//^^
//{[{

        // TODO WTS: Remove this once your ListDetail pages are displaying real data.
        public async Task<IEnumerable<SampleOrder>> GetListDetailDataAsync()
        {
            await Task.CompletedTask;
            return AllOrders();
        }
//}]}
    }
}

File: templates/Uwp/_comp/_shared/Page.ListDetail.SampleData._VB/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the method `GetListDetailDataAsync()` to the SampleDataService of your project.
'**}
Namespace Services
    Public Module SampleDataService
'^^
'{[{

        ' Remove this once your ListDetail pages are displaying real data.
        Public Async Function GetListDetailDataAsync() As Task(Of IEnumerable(Of SampleOrder))
            Await Task.CompletedTask

            Return AllOrders()
        End Function
'}]}
    End Module
End Namespace
